common/customizable_localization/00_knight_culture.txt

```diff
@@ -1,287 +1,749 @@
 ﻿KnightCulture = {
 	type = character
-
+	
+	#elf group#
 	text = {
 		trigger = {
-			OR = {
-				has_culture_group = culture_group:hurrian_group
-				has_culture_group = culture_group:zagrosian_group
-				has_culture_group = culture_group:levantine_group
-				has_culture_group = culture_group:egyptian_group
-				has_culture_group = culture_group:akkadian_group
-				has_culture_group = culture_group:elamite_group
-				has_culture_group = culture_group:luwian_group
+			OR{
+				has_culture = culture:night_elf
+				has_culture = culture:high_elf
+				has_culture = culture:highborne
+				has_culture = culture:nightborne
 			}
-			culture = { has_innovation = innovation_spoked_wheel }
+			
+		}
+		localization_key = knight_Warden
+	}
+	text = {
+		trigger = {
+			has_culture = culture:blood_elf
+		}
+		localization_key = knight_BloodKnights
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:orc_group
+		}
+		localization_key = knight_Blademaster
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:troll_group
+		}
+		localization_key = knight_Berserker
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:dwarf_group
+			
+		}
+		localization_key = knight_Thane
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:gnome_group
 		}
-		localization_key = knight_maryannu
+		localization_key = knight_Gearmaster
 	}
 	text = {
 		trigger = {
-			has_culture_group = culture_group:hellenic_group
+			has_culture_group = culture_group:demonic_group
 		}
-		localization_key = knight_hekwetai
+		localization_key = knight_Doomguard
 	}
 	text = {
 		trigger = {
-			culture = { has_innovation = innovation_spoked_wheel }
+			has_culture_group = culture_group:tauren_group
 		}
-		localization_key = knight_charioteer
+		localization_key = knight_Runemaster
 	}
+	
+	#Undead Group#
+	text = {
+		trigger = {
+			has_culture = culture:scourge
+		}
+		localization_key = knight_Deathknights
+	}
+	text = {
+		trigger = {
+			has_culture = culture:forsaken
+		}
+		localization_key = knight_Dreadguard
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:draenei_group
+		}
+		localization_key = knight_Vindicator
+	}
+	text = {
+		trigger = {
+			has_culture_group = culture_group:pandaren_group
+		}
+		localization_key =  knight_Shodo-pan
+	}
+	
+	text = {
+		trigger = {
+			has_culture_group = culture_group:human_group
+		}
+		localization_key = knight_default
+	}
+
+	
 	text = {
-		localization_key = knight_hero
+		localization_key = knight_Champion
```

Comment:
`knight_champion` is vanilla localization thus there's no need to define it twice. Just use link vanilla lines.

Reply:
Is it really resolved though?